doc/compiled.json

```diff
@@ -26575,6 +26622,207 @@
         ],
         "x-cli-version": "2.7.0"
       }
+    },
+    "/projects/{project_id}/keys/{id}/key_links": {
+      "delete": {
+        "summary": "Batch unlink child keys from a parent key",
+        "description": "Unlinks multiple child keys from a given parent key in a single operation.",
+        "operationId": "linked_keys/batch_destroy",
```

Comment:
If the endpoint is called `key_links` shouldn't the operation id be called `key_links` too?

Reply:
Thx!